cg/apps/demultiplex/sample_sheet/models.py

```diff
@@ -14,12 +17,18 @@
 
 LOG = logging.getLogger(__name__)
 
+SampleId = Annotated[
```

Comment:
Should this be in a `cg/apps/demultiplex/sample_sheet/validators.py` and imported here? It feels a bit out of place defined directly in this module